index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <body style="background-color:lightgray;"></body>  
+  <link rel="stylesheet" href="css/style.css">
+  <head>
+    <h1 id= "title"> A Tribute to Micheal Jackson </h1>
+    <p>King of Pop</p>
+    
+    <main id= main>
+    <meta charset="UTF-8">
+    <title>Raf & Boku project</title>
+    
+  </head>
+  <figure id="img-div">
+<img id= "image" class="center" src="https://www.billboard.com/wp-content/uploads/stylus/2528869-michael-jackson-617-409.jpg?w=617&h=409&crop=1
+">
+<style type="text/css">
+    img {
+  border: 50px solid rgb(255, 255, 255);
+}
+    h1 {text-align: center;}
+    h2 {text-align: center;}
+    div {text-align: center;}
+    p {text-align: center;}
+        .center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 70%;
+  }
+    </style>
+</figure>
+
+<div id="img-caption">
+  <strong>Michael Joseph Jackson</strong>
```

Comment:
It would be a much better idea to use h2 tag for this text. (OPTIONAL)

Reply:
As I suggested above explain why you are suggesting the changes to a student so that a student would understand the reason he/she would implement those changes.